types/expressions.d.ts

```diff
@@ -2381,31 +2383,41 @@ declare module 'mongoose' {
         lang: 'js'
       };
     }
-  }
 
-  type Path = string;
+    export interface Exists {
+      /**
+        * When <boolean> is true, $exists matches the documents that contain the field, including documents where the field value is null.
+        * If <boolean> is false, the query returns only the documents that do not contain the field.
+        *
+        * @see https://www.mongodb.com/docs/manual/reference/operator/query/exists/
+        */
+      $exists: boolean;
+    }
+  }
 
-  export type Expression =
-    Path |
-    ArithmeticExpressionOperator |
-    ArrayExpressionOperator |
-    BooleanExpressionOperator |
-    ComparisonExpressionOperator |
-    ConditionalExpressionOperator |
-    CustomAggregationExpressionOperator |
-    DataSizeOperator |
-    DateExpressionOperator |
-    LiteralExpressionOperator |
-    MiscellaneousExpressionOperator |
-    ObjectExpressionOperator |
-    SetExpressionOperator |
-    StringExpressionOperator |
-    TextExpressionOperator |
-    TrigonometryExpressionOperator |
-    TypeExpressionOperator |
-    AccumulatorOperator |
-    VariableExpressionOperator |
-    WindowOperator;
+  type Path = string | ObjectIdLike;
+
+
+  export type Expression = Path |
+  ArithmeticExpressionOperator |
```

Comment:
<img width="530" alt="Screenshot 2022-06-20 at 11 48 38" src="https://user-images.githubusercontent.com/697707/174563205-20b99ce1-306a-4b8a-b984-e42c1bff141b.png">

Not sure why this got reformatted, but I'm getting this otherwise.